distribution/kubeflow/katib/kustomization.yaml

```diff
@@ -1,8 +1,16 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+namespace: kubeflow
 
 resources:
-- github.com/kubeflow/katib/manifests/v1beta1/installs/katib-with-kubeflow?ref=release-0.11
+- github.com/kubeflow/katib/manifests/v1beta1/installs/katib-external-db?ref=release-0.11
```

Comment:
I believe this line shouldn't have been changed.

Reply:
right!